### PR TITLE
Add organization to hearing

### PR DESCRIPTION
**Pull request by Rikuoja**
Required by #138

### CODE REVIEW

File: democracy/models/base.py [hunk 1]
```diff
@@ -116,6 +118,13 @@ class Meta:
         abstract = True
 
 
+class Organization(BaseModel):
+    name = models.CharField(verbose_name=_('name'), max_length=255)
+    admin_users = models.ManyToManyField(
+        User, blank=True, related_name='admin_organizations'
```
Comment (akx) [inline democracy/models/base.py]: Just use `settings.AUTH_USER_MODEL` directly here -- aliasing it to `User` makes the casual reader think it's not pluggable.

**Comment by akx** [comment]
Missing tests.

What is the actual desired serialization format for `organization`? (Should it maybe be a `SlugRelatedField`?)

**Comment by akx** [comment]
Also missing a migration.

File: democracy/migrations/0012_auto_20160502_1219.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
Comment (akx) [inline democracy/migrations/0012_auto_20160502_1219.py]: Please use the `-n` flag with `makemigrations` to specify the migration's name. (`auto` migration names are not very nice.)

File: democracy/migrations/0013_auto_20160502_1222.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (akx) [inline democracy/migrations/0013_auto_20160502_1222.py]: Please use the `-n` flag with `makemigrations` to specify the migration's name. (`auto` migration names are not very nice.)

File: democracy/models/label.py [hunk 4]
```diff
@@ -1,10 +1,10 @@
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 
-from .base import BaseModel
+from .base import StringIdBaseModel
 
 
-class Label(BaseModel):
+class Label(StringIdBaseModel):
```
Comment (akx) [inline democracy/models/label.py]: This seems like an unrelated change to "Add organization to hearing"?

Comment (akx) [inline democracy/models/label.py]: And all in all, I'm not quite sure what this change is supposed to achieve. The commit message is quite opaque (and lacks any sort of issue ref).

Comment (Rikuoja) [inline democracy/models/label.py]: Agreed, it is only tangentially related. I created issue #171 for the change.

Comment (akx) [inline democracy/models/label.py]: Please add a reference to #171 in the commit message too.

**Comment by akx** [comment]
@Rikuoja Rebase required.

**Comment by akx** [comment]
Tossing the ball to @tuomas777 to review + merge.

File: democracy/models/base.py [hunk 5]
```diff
@@ -115,6 +115,13 @@ class Meta:
         abstract = True
 
 
+class Organization(StringIdBaseModel):
+    name = models.CharField(verbose_name=_('name'), max_length=255, unique=True)
+    admin_users = models.ManyToManyField(
+        settings.AUTH_USER_MODEL, blank=True, related_name='admin_organizations'
+    )
```
Comment (tuomas777) [inline democracy/models/base.py]: IMO this model shouldn't be in `base.py` as it isn't used as a basis for anything

File: democracy/tests/test_hearing.py [hunk 6]
```diff
@@ -214,6 +215,25 @@ def test_8_get_detail_labels(api_client):
 
 
 @pytest.mark.django_db
+def test_8_get_detail_organization(api_client):
+    hearing = Hearing()
+    hearing.save()
+
+    organization = Organization(name='The department for squirrel welfare')
+    organization.save()
```
Comment (tuomas777) [inline democracy/tests/test_hearing.py]: You could use pytest fixture for hearing here, there is `default_hearing` for example. Also organization should be made as a fixture.

File: democracy/models/base.py (hunk 5 above)
Comment (Rikuoja) [inline democracy/models/base.py]: Ah, didn't realize that was the point :)

File: democracy/migrations/0015_add_organization_model.py [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+    operations = [
+        migrations.CreateModel(
+            name='Organization',
+            fields=[
+                ('created_at', models.DateTimeField(db_index=True, default=django.utils.timezone.now, editable=False, verbose_name='time of creation')),
+                ('modified_at', models.DateTimeField(default=django.utils.timezone.now, editable=False, verbose_name='time of last modification')),
+                ('published', models.BooleanField(db_index=True, default=True, verbose_name='public')),
+                ('deleted', models.BooleanField(db_index=True, default=False, editable=False, verbose_name='deleted')),
+                ('id', models.CharField(blank=True, help_text='You may leave this empty to automatically generate an ID', max_length=32, primary_key=True, serialize=False, verbose_name='identifier')),
```
Comment (tuomas777) [inline democracy/migrations/0015_add_organization_model.py]: Actually this is a bit wrong because `StringIdBaseModel` has changed, I *think* correct would be `('id', models.CharField(editable=False, max_length=32, primary_key=True, serialize=False, verbose_name='identifier'))`

**Comment by tuomas777** [comment]
LGTM!
